src/index.ts

```diff
@@ -48,6 +48,59 @@ export interface GetSubscriptionMetadataCallback {
   (err: ServiceError|null, res?: google.pubsub.v1.Subscription|null): void;
 }
 
+export interface Attributes {
+  [key: string]: string;
+}
+
+export interface ClientConfig {
+  projectId?: string;
+  keyFilename?: string;
+  apiEndpoint?: string;
+  email?: string;
+  credentials?: {client_email: string, private_key: string};
+  autoRetry?: boolean;
+  maxRetries?: number;
+  promise?: PromiseConstructor;
+  servicePath?: string;
+  port?: string;
+  sslCreds?: ChannelCredentials;
```

Comment:
why do you need sslcred and credential both? I dont think you need sslcreds.

Reply:
at 297, they mention in comment structure of ClientConfig, there they mention it, so i used, its not used anywhere in application, so removed!